Development/src/pages/logs.js

```diff
@@ -174,20 +163,14 @@ export const EventsList = () => {
                     <PaginationButton label="PREV" nextPage={nextPage} />
                     <PaginationButton label="NEXT" nextPage={nextPage} />
                     <PaginationButton label="LAST" nextPage={nextPage} />
-                    <Button
```

Comment:
So you decided that with the Refresh button working again, we might as well get rid of Clear All Filters everywhere?

Reply:
Unfortunately I was unable to find a way to make Clear All Filters correctly function in the time frame I had. This is because the `<FilterField>` component stores it's filter value in it's own state. As child props cannot be modified from a parent component, the only way to refresh the filter components is to reload the whole page. In the future, filter state will have to be managed by the Parent for this to work

Reply:
I don't think it's necessary, honestly.